demos/01 Unsupervised Classification/01c Interactive Deployment with Dynamic Compute.ipynb

```diff
@@ -172,7 +172,13 @@
     "    bands,\n",
     "    start_datetime=\"2023-06-01\",\n",
     "    end_datetime=\"2023-09-01\",\n",
-    ").filter(lambda x: x.cloud_fraction < 0.1)"
+    ")\n",
+    "if float(dc.__version__[2]) >= 3:\n",
```

Comment:
sorry, these need to be `float(dc.__version__[:3]) >= 1.3` , `rather than float(dc.__version__[2]) >= 3`